app/models/timdex.rb

```diff
@@ -0,0 +1,70 @@
+require 'graphql/client'
+require 'graphql/client/http'
+
+class Timdex
+  HTTP = GraphQL::Client::HTTP.new(ENV.fetch('TIMDEX_GRAPHQL', '')) do
+    def headers(*)
+      { 'User-Agent': 'MIT Libraries Client' }
+    end
+  end
+
+  Schema = GraphQL::Client.load_schema('config/schema/schema.json')
```

Comment:
Can you document how to regenerate this file in the README and provide a bit of context as to when it needs to be regenerated (which I think is largely whenever we update the the GraphQL schema upstream)

Reply:
Oh yeah - that's a good suggestion. Second commit underway...